### PR TITLE
CB-14482 Set the callingServiceName GRPC context for UMS calls

### PR DESCRIPTION
**Pull request by foldik**
See detailed description in the commit message.

### CODE REVIEW

File: auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/config/UmsClientConfig.java [hunk 1]
```diff
@@ -23,6 +23,9 @@
     @Value("${altus.ums.client.list_service_principal_cloud_identities_page_size:100}")
     private int listServicePrincipalCloudIdentitiesPageSize;
 
+    @Value("${altus.ums.calling_service_name:cloudbreak}")
```
Comment (horadla23) [inline auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/config/UmsClientConfig.java:26]: spring property with underscores looks strange, I would use `caller.service.name` or simply `caller`, what do you think about it?

Comment (foldik) [reply auth-connector/src/main/java/com/sequenceiq/cloudbreak/auth/altus/config/UmsClientConfig.java]: Renamed the property to `caller`.

**Review by horadla23 — APPROVED** [review]

**Review by lnardai — APPROVED** [review]
Can't we add an assertion that ensures the header added properly?

**Comment by foldik** [comment]
@lnardai I can add one to mock ums.

**Review by doktoric — APPROVED** [review]

**Review by akanto — APPROVED** [review]

**Pull request merged**